README.rst

```diff
@@ -2,93 +2,103 @@
-   :target: https://zenodo.org/badge/latestdoi/2165383
+    :target: https://zenodo.org/badge/latestdoi/2165383
+    :alt: Zenodo - Latest DOI
 .. |matrix| image:: https://img.shields.io/matrix/sunpy:openastronomy.org.svg?colorB=%23FE7900&label=Chat&logo=matrix&server_fqdn=matrix.org
-   :target: https://app.element.io/#/room/#sunpy:openastronomy.org
+    :target: https://app.element.io/#/room/#sunpy:openastronomy.org
+    :alt: Matrix Chat Room
 .. |codecov| image:: https://codecov.io/gh/sunpy/sunpy/branch/main/graph/badge.svg
-   :target: https://codecov.io/gh/sunpy/sunpy
+    :target: https://codecov.io/gh/sunpy/sunpy
+    :alt: Code Coverage
 .. |Powered by NumFOCUS| image:: https://img.shields.io/badge/powered%20by-NumFOCUS-orange.svg?style=flat&colorA=E1523D&colorB=007D8A
-   :target: https://numfocus.org
+    :target: https://numfocus.org
+    :alt: Powered by NumFOCUS
 .. |CI| image:: https://github.com/sunpy/sunpy/actions/workflows/ci.yml/badge.svg?branch=main
-   :target: https://github.com/sunpy/sunpy/actions/workflows/ci.yml
+    :target: https://github.com/sunpy/sunpy/actions/workflows/ci.yml
+    :alt: Continuous Integration Status
 .. |Docs| image:: https://readthedocs.org/projects/sunpy/badge/?version=stable
-   :target: https://docs.sunpy.org/en/stable/?badge=stable
-   :alt: Documentation Status
+    :target: https://docs.sunpy.org/en/stable/?badge=stable
+    :alt: Stable Documentation
 .. |repostatus| image:: https://www.repostatus.org/badges/latest/active.svg
-   :alt: Project Status: Active – The project has reached a stable, usable state and is being actively developed.
-   :target: https://www.repostatus.org/#active
+    :target: https://www.repostatus.org/#active
+    :alt: Project Status: Active - The project has reached a stable, usable state and is being actively developed.
 .. |python| image:: https://img.shields.io/pypi/pyversions/sunpy
-   :alt: PyPI - Python Version
+    :alt: PyPI - Python Version
 
 ``sunpy`` is a Python software package that provides fundamental tools for accessing, loading and interacting with solar physics data in Python.
-It is the core library of the `SunPy Project <https://sunpy.org/>`__.
+It includes an interface for searching and downloading data from multiple data providers, data containers for image and time series data, commonly used solar coordinate frames and associated transformations, as well as other functionality needed for solar data analysis.
 
-For some examples of using ``sunpy`` see our `gallery <https://docs.sunpy.org/en/stable/generated/gallery/index.html>`__.
-To see the latest changes in ``sunpy`` see our `changelog <https://docs.sunpy.org/en/stable/whatsnew/changelog.html>`__.
+It is the core library of the `SunPy Project <https://sunpy.org/>`__ that forms the foundation of the ecosystem of affiliated packages that aims to be a community-developed, free and open-source solar data analysis environment for Python.
+This is the mission statement of the SunPy project.
```

Comment:
This sentence feels... odd? Is it here to meet a PyOpenSci requirement?

Reply:
Now, I just copied this from the website. I can undo it.

Reply:
Yeah I'm not a fan of this. I also don't think we ever agreed on a mission statement...

Reply:
Removed.